### PR TITLE
tidb: add explanation for the fields in mysql.user

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

* add explanation for the fields in `mysql.user`
* put `mysql-schema*.md` in a unique dir

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/18318
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: mysql-schema/mysql-schema-user.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title：`mysql.user`
+summary：了解 `mysql` 系统表中的 `user` 表。
```
Comment (CbcWestwolf) [inline mysql-schema/mysql-schema-user.md:3]:
```suggestion
summary：了解 `mysql` 系统表 `user` 。
```

File: mysql-schema/mysql-schema-user.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+| Password_expired       | enum('N','Y')        | NO   |      | N                 |       |
+| Password_last_changed  | timestamp            | YES  |      | CURRENT_TIMESTAMP |       |
+| Password_lifetime      | smallint(5) unsigned | YES  |      | NULL              |       |
++------------------------+----------------------+------+------+-------------------+-------+
+44 rows in set (0.00 sec)
+```
+
+`mysql.user` 表包含多个字段，可分为三组：
+
+* 作用域相关字段：
```
Comment (CbcWestwolf) [inline mysql-schema/mysql-schema-user.md:72]:
```suggestion
* 账户相关字段：
```

**Review by CbcWestwolf — APPROVED** [review]

File: mysql-schema/mysql-schema-user.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
+---
+title：`mysql.user`
+summary：了解 `mysql` 系统表 `user` 。
```
Comment (lilin90) [inline mysql-schema/mysql-schema-user.md:3]:
```suggestion
title: `mysql.user`
summary: 了解 `mysql` 系统表 `user`。
```

File: mysql-schema/mysql-schema.md [hunk 4]
```diff
@@ -8,11 +8,15 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
 
 `mysql` 库里存储的是 TiDB 系统表。该设计类似于 MySQL 中的 `mysql` 库，其中 `mysql.user` 之类的表可以直接编辑。该库还包含许多 MySQL 的扩展表。
 
+> **注意：**
+>
+> 在大多数情况下，不建议直接使用 `INSERT`、`UPDATE` 或 `DELETE` 更改系统表的内容。相反，建议使用 [`CREATE USER`](/sql-statements/sql-statement-create-user.md)、[`ALTER USER`](/sql-statements/sql-statement-alter-user.md)、[`DROP USER`](/sql-statements/sql-statement-drop-user.md)、[`GRANT`](/sql-statements/sql-statement-grant-privileges.md)、[`REVOKE`](/sql-statements/sql-statement-revoke-privileges.md) 和 [`SHOW CREATE USER`](/sql-statements/sql-statement-show-create-user.md) 等语句管理用户和权限。如果必须直接修改系统表，请使用 [`FLUSH PRIVILEGES`](/sql-statements/sql-statement-flush-privileges.md) 使更改生效。
```
Comment (lilin90) [inline mysql-schema/mysql-schema.md:13]:
```suggestion
> 在大多数情况下，不建议直接使用 `INSERT`、`UPDATE` 或 `DELETE` 更改系统表的内容。而是建议使用 [`CREATE USER`](/sql-statements/sql-statement-create-user.md)、[`ALTER USER`](/sql-statements/sql-statement-alter-user.md)、[`DROP USER`](/sql-statements/sql-statement-drop-user.md)、[`GRANT`](/sql-statements/sql-statement-grant-privileges.md)、[`REVOKE`](/sql-statements/sql-statement-revoke-privileges.md) 和 [`SHOW CREATE USER`](/sql-statements/sql-statement-show-create-user.md) 等语句管理用户和权限。如果必须直接修改系统表，请使用 [`FLUSH PRIVILEGES`](/sql-statements/sql-statement-flush-privileges.md) 使更改生效。
```

**Review by lilin90 — APPROVED** [review]
Rest LGTM

**Pull request merged**